ql/termstructures/volatility/swaption/sabrswaptionvolatilitycube.hpp

```diff
@@ -1151,16 +1157,30 @@ namespace QuantLib {
         return result;
     }
 
+    /*! \deprecated Use XabrSwaptionVolatilityCube instead.
+                    Deprecated in version 1.29.
+    */    
+
```

Comment:
As you've seen, `typedef` doesn't work with templates but you can use the syntax:
```
template <class Model>
using SwaptionVolCube1x = XabrSwaptionVolatilityCube<Model>
```

Reply:
It seems I forgot the `;` at the end...